blurup.js

```diff
@@ -34,10 +34,11 @@ export async function createBlurUp(playbackId, options) {
 
   quality = parseFloat(quality);
 
-  if (quality >= 1) {
-    imageURL.searchParams.set('width', 16 * quality);
-    imageURL.searchParams.set('height', 16 * quality);
+  if (isNaN(quality) || quality <= 0) {
```

Comment:
a little strange but `1` should be really the minimum value. I think that was the goal. because we want 16px to be the smallest side possible. below 16px and the compression makes the thumb too bad. I believe it also has to do with WebP encoding in 16x16 blocks.

Reply:
Thanks for clarifying! I still think we should error if the number is below 1. That could be out of scope for this PR though, if you prefer.

Reply:
yep, I like that. if it's less than 1 throw an error 👍